RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
OK, then staying with *Tmp now looks like the right strategy
[the relinking step will introduce new products reading these *Tmp
and making non-Tmp ones
]

		

On 8/12/15 10:55 AM, mandrenguyen wrote:
> In RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py
> <https://github.com/cms-sw/cmssw/pull/10700#discussion_r36877049>:
> 
>> @@ -24,7 +26,9 @@
>>      "drop recoPFClusters_*_*_*",
>>      "keep recoElectronSeeds_*_*_*",
>>      "keep recoGsfElectrons_*_*_*",
>> -    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
>> +    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
>> +    'keep recoPhotons_gedPhotonsTmp_*_*',
> 
> Ok, that sounds like something we should fix for the heavyIon scenario then.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10700/files#r36877049>.
>